src/sentry/api/endpoints/project_details.py

```diff
@@ -418,9 +418,7 @@ def put(self, request: Request, project) -> Response:
         :param int digestsMaxDelay:
         :auth: required
         """
-        has_project_write = (request.auth and request.auth.has_scope("project:write")) or (
-            request.access and request.access.has_scope("project:write")
-        )
+        has_project_write = request.access and request.access.has_scope("project:write")
```

Comment:
Does this have the potential to break any existing integrations/scripts?

If so we should measure the potential impact.